### PR TITLE
feature: exposing headers for mircommon-internal-dev

### PR DESCRIPTION
**Pull request by mattkae**
## What's new?
- Generated internal mircommon symbols with the new script
- Created `libmircommon-internal-dev` to include internal common headers

## To use:
```cmake
pkg_check_modules(MIRCOMMON_INTERNAL mircommon-internal REQUIRED)
target_include_directories(${BINARY_NAME} PUBLIC SYSTEM ... ${MIRCOMMON_INTERNAL_INCLUDE_DIRS})
```

### CODE REVIEW

**Review by AlanGriffiths — CHANGES_REQUESTED** [review]
"Applications on the remote desktop can be run on the client" seems an odd commit comment for this PR. And I don't see why `run.sh` is added here? (Forgotten in an earlier PR perhaps?)

Anyway, needs clarification at least

File: tools/symbols_map_generator/run.sh
[the remark this answers is not in the document]
Comment (mattkae) [reply tools/symbols_map_generator/run.sh:1]: Whoops!

**Comment by mattkae** [comment]
> "Applications on the remote desktop can be run on the client" seems an odd commit comment for this PR. And I don't see why `run.sh` is added here? (Forgotten in an earlier PR perhaps?)
> 
> Anyway, needs clarification at least

Multitasking, wrong copy and paste XD `run.sh` and `requirements.txt` were a bad merge

File: debian/control [hunk 1]
```diff
@@ -107,7 +107,7 @@ Section: libdevel
 Architecture: linux-any
 Multi-Arch: same
 Pre-Depends: ${misc:Pre-Depends}
-Depends: libmircommon10 (= ${binary:Version}),
+Depends: libmircommon11 (= ${binary:Version}),
```
Comment (AlanGriffiths) [inline debian/control:110]: Hmm, `debian/changelog` for 2.16 says "mircommon ABI unchanged at 9". And...

```
$ git diff v2.16.4 -- debian/libmircommon10.install
diff --git a/debian/libmircommon10.install b/debian/libmircommon10.install
new file mode 100644
index 0000000000..5233fdfb45
--- /dev/null
+++ b/debian/libmircommon10.install
@@ -0,0 +1 @@
+usr/lib/*/libmircommon.so.10
```

So 10 is already new in 2.17

**Review by AlanGriffiths — CHANGES_REQUESTED** [review]
looks fine apart from the spurious .soname bump

**Review by AlanGriffiths — APPROVED** [review]

**Pull request merged**